certificatetprtest/service.go

```diff
@@ -0,0 +1,18 @@
+package certificatetprtest
```

Comment:
Interesting idea but I do not see a need for it now. Lets iterate here and see how it goes.